opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -495,6 +498,24 @@ public void hideProgressDialog() {
         }
     }
 
+    @Override
+    public void getUserCurrentLocation() {
```

Comment:
Is this the best place to have this kind of logic?

Why not have it in the `ListTasksPresenter` similar to what `RevealJsonFormFragmentPresenter` has?

The view should avoid adding logic, right?